metadata-ingestion/src/datahub/ingestion/source_report/pulsar.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass, field
+from typing import List
+
+from datahub.ingestion.source.state.stateful_ingestion_base import (
+    StatefulIngestionReport,
+)
+
+
+@dataclass
+class PulsarSourceReport(StatefulIngestionReport):
+    pulsar_version: str = ""
+    tenants_scanned: int = 0
```

Comment:
`Optional[int]` instead of `int`? This might cause some lint failures and needing to set them to 0. But helps to see the point where execution went to. Same comment for other `int` in the report.